### ISSUE TITLE
failing tests on master

### ISSUE THREAD

**Issue by maxogden** [maxogden/standard-format#14]
output is below. are these supposed to be failing? if so, lets try and keep failing tests out of master in the future

```
🐈  npm test

> standard-format@1.2.0 test /Users/maxogden/src/js/standard-format
> standard && tape test/*.js

TAP version 13
# comments formatting
not ok 1 Expect space or tab after // in comment
  ---
    operator: equal
    expected: '// bad comment\n'
    actual:   '//bad comment\n'
    at: Test.<anonymous> (/Users/maxogden/src/js/standard-format/test/comments.js:10:5)
  ...
ok 2 Expect good comments to be unchanged
# test.js ran through formatter
ok 3 no runner errors
not ok 4 Multiple blank lines not allowed.
  ---
    operator: fail
    at: f (/Users/maxogden/src/js/standard-format/node_modules/once/once.js:17:25)
  ...

stdin:1:0:Multiple blank lines not allowed.

^
not ok 5 Unexpected space between function name and paren.
  ---
    operator: fail
    at: f (/Users/maxogden/src/js/standard-format/node_modules/once/once.js:17:25)
  ...

stdin:14:0:Unexpected space between function name and paren.
console.log(x)
^
not ok 6 Expected space or tab after // in comment.
  ---
    operator: fail
    at: f (/Users/maxogden/src/js/standard-format/node_modules/once/once.js:17:25)
  ...

stdin:21:0:Expected space or tab after // in comment.
//bad comment -- needs a space after slashes
^
not ok 7 test is defined but never used
  ---
    operator: fail
    at: f (/Users/maxogden/src/js/standard-format/node_modules/once/once.js:17:25)
  ...

stdin:22:4:test is defined but never used
var test = 'what'
    ^
not ok 8 Unexpected space between function name and paren.
  ---
    operator: fail
    at: f (/Users/maxogden/src/js/standard-format/node_modules/once/once.js:17:25)
  ...

stdin:30:0:Unexpected space between function name and paren.
(function f2 () {})()
^
# multiline 
ok 9 two empty lines should collapse to one
ok 10 ten empty lines should collapse to one
ok 11 single empty line should be unmodified

1..11
# tests 11
# pass  5
# fail  6

npm ERR! Test failed.  See above for more details.
```

**Comment by jb55** [comment]
you want to remove the tests from master?

**Comment by maxogden** [comment]
@jb55 if they are supposed to be failing, then yea they should go on a branch. IMO we should keep master passing always

**Comment by jb55** [comment]
fixed in ccb555f
